src/modules/Crop/Module.js

```diff
@@ -13,21 +13,23 @@
  *          y = options.y
  *          y = options.y + options.h
  */
- module.exports = function CropModule(options) {
+ module.exports = function CropModule(options,UI) {
    options = options || {};
    options.title = "Crop Image";
-   var this_ = this;
+   UI.onSetup();
    var output
 
    function draw(input,callback) {
 
+     UI.onDraw();
      const this_ = this;
 
      require('./Crop')(input,options,function(out,format){
        this_.output = {
```

Comment:
Yes, you are right...
I had actually gotten into the practice of doing this from writing callbacks (and avoiding the scope change problem of `this`):
```js
function x() {
    var y = 1;
    var this_ = this;
    someObj.someMethod(abc, function() {
        this_.y = 2;
    });
}
```

I will change it. Thanks!